src/server/index.js

```diff
@@ -43,7 +45,7 @@ const formatError = (err) => {
 const getEnvelop = async () =>
   envelop({
     plugins: [
-      useSchema(await apiSchema),
+      useAsyncSchema(apiSchema),
```

Comment:
is it working now? Some time ago I tried using it but it didn't work really well

Reply:
yeah, it is.